<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>871</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>871</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mancala</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>850</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236,236,236)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QLabel" name="alertLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>851</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="endGame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>End Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startGame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="store2Label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="store1Label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="instructionsFrame">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>40</y>
      <width>611</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="instructionsCover">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>351</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255,255,255,240)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>25</number>
     </property>
    </widget>
    <widget class="QLabel" name="instuctionsTitle">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>15</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Instructions</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="i_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>531</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>- Each player starts with 4 marbles in each of their 6 pits.</string>
     </property>
    </widget>
    <widget class="QLabel" name="i_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>531</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>- When a pit is pressed the marbles in that pit get distributed into the next pits.</string>
     </property>
    </widget>
    <widget class="QLabel" name="i_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>531</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>- The goal is to get as many marbles into your marble store as possible</string>
     </property>
    </widget>
    <widget class="QLabel" name="i_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>531</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>- Each player has a marble store, which are the larger pits at the ends of the board</string>
     </property>
    </widget>
    <widget class="QLabel" name="i_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>**Stealing Marbles:** If your last marble lands in an empty pit or a **special pit** you steal the marbles from the pit across</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="i_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>**Extra turns:** When marbles are being distributed, if your last marble lands in your marble store, then you get to take another turn</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="i_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>**Special Marbles:** Worth 5 points towards your overall score, where regular marbles are worth 1 point</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="i_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>**Special Pit:** Each player has a special pit, if your last marble lands in this pit you automatically steal the marbles from across, and one is upgraded to a special marble</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
